### PR TITLE
SK-1473-WEB-Email-alloffers-manage_redirections_alias

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md [hunk 1]
```diff
@@ -32,17 +34,32 @@ Par exemple, votre adresse e-mail est **john.smith@mydomain.ovh** et votre alias
 
 ### Redirection et alias en image <a name="diagram"></a>
 
-- **La redirection simple (schéma n°1 ci-dessous)** : l'e-mail est directement renvoyé vers l'adresse de redirection, le destinataire initial ne reçoit pas l'e-mail.
-
-- **La redirection avec copie locale (schéma n°2 ci-dessous)** : l'e-mail est transmis au destinataire initial ainsi qu'à l'adresse de redirection.
+Cliquez sur les onglets suivants pour comprendre en image le fonctionnement des alias et redirections.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md]:
```suggestion
Cliquez sur les onglets suivants pour des explications illustrées sur le fonctionnement des alias et redirections.
```

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md [hunk 2]
```diff
@@ -32,17 +34,32 @@ Par exemple, votre adresse e-mail est **john.smith@mydomain.ovh** et votre alias
...
-- **L'alias e-mail (schéma n°3 ci-dessous)** : l'e-mail est adressé à l'alias qui le renvoie vers le destinataire sur lequel l'alias a été configuré.
+- `From` désigne l'adresse de l'expéditeur
+- `To` désigne l'adresse du destinataire
+- `Redirect to` désigne l'adresse e-mail de redirection qui a été configurée.
 
-![emails](images/schema-redirect.png){.thumbnail}
+> [!tabs]
+> **1. La redirection simple**
+>>
+>> l'e-mail est directement renvoyé vers l'adresse de redirection, le destinataire initial ne reçoit pas l'e-mail.<br><br>
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md]:
```suggestion
>> L'e-mail est directement renvoyé vers l'adresse de redirection, le destinataire initial ne reçoit pas l'e-mail.<br><br>
```

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md [hunk 3]
```diff
@@ -32,17 +34,32 @@ Par exemple, votre adresse e-mail est **john.smith@mydomain.ovh** et votre alias
...
+>> ![emails](images/schema-redirect01.png){.thumbnail}
+>>
+> **2. La redirection avec copie locale**
+>>
+>> l'e-mail est transmis au destinataire initial ainsi qu'à l'adresse de redirection.<br><br>
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md]:
```suggestion
>> L'e-mail est transmis au destinataire initial ainsi qu'à l'adresse de redirection.<br><br>
```

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md [hunk 4]
```diff
@@ -32,17 +34,32 @@ Par exemple, votre adresse e-mail est **john.smith@mydomain.ovh** et votre alias
...
+>> ![emails](images/schema-redirect02.png){.thumbnail}
+>>
+> **3. L'alias e-mail**
+>>
+>> l'e-mail est adressé à l'alias qui le renvoie vers le destinataire sur lequel l'alias a été configuré. La mention `Received by` désigne l'adresse e-mail qui reçoit l'e-mail.<br><br>
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md]:
```suggestion
>> L'e-mail est adressé à l'alias qui le renvoie vers le destinataire sur lequel l'alias a été configuré. La mention `Received by` désigne l'adresse e-mail qui reçoit l'e-mail.<br><br>
```

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md [hunk 5]
```diff
@@ -32,17 +34,32 @@ Par exemple, votre adresse e-mail est **john.smith@mydomain.ovh** et votre alias
...
+>> ![emails](images/schema-redirect03.png){.thumbnail}
+>>
 
 > [!primary]
 >
-> Sachez qu'il est possible de configurer une redirection vers plusieurs adresses e-mail.
+> Sachez qu'il est possible de configurer une redirection vers plusieurs adresses e-mail. Pour cela, il est néanmoins nécessaire de créer une par une les redirections vers chaque destinataire.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md]:
```suggestion
> Sachez qu'il est possible de configurer une redirection vers plusieurs adresses e-mail. Cela implique cependant de créer une par une les redirections vers chaque destinataire.
```

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md [hunk 6]
```diff
@@ -51,12 +68,15 @@ Par exemple, votre adresse e-mail est **john.smith@mydomain.ovh** et votre alias
...
-
-- vous possédez la version historique de l'offre MXplan (pour le vérifier, vous pouvez consulter le cadre ci-dessous).
-- vous possédez un nom de domaine enregistré chez OVHcloud sans offre e-mail attachée.
-
-Pour ces deux exceptions, seule la redirection est possible. Nous vous invitons à consulter directement le chapitre « [Créer une redirection ou un alias sur une offre MX Plan historique ou un nom de domaine sans offre e-mail](#mxplanlegacy) ».
+> [!warning]
+>
+> Les chapitres « [créer une redirection](#redirect) » et « [créer un alias](#alias) » concernent l'ensemble de nos offres e-mail OVHcloud, **sauf si** :
+> 
+> - vous possédez la version historique de l'offre MXplan (pour le vérifier, vous pouvez consulter le cadre ci-dessous).
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md]:
```suggestion
> - vous possédez la version historique de l'offre MX Plan (pour le vérifier, vous pouvez consulter le cadre ci-dessous).
```

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md [hunk 7]
```diff
@@ -70,7 +90,7 @@ Pour ces deux exceptions, seule la redirection est possible. Nous vous invitons
 
 ### Créer une redirection <a name="redirect"></a>
 
-La gestion des redirections ne se fait pas via l'espace client mais directement via le webmail de l'adresse e-mail concernée.
+La gestion des redirections ne se fait pas via l'espace client, mais directement via le webmail de l'adresse e-mail concernée.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md]:
```suggestion
La gestion des redirections ne se fait pas via l'espace client mais directement via le webmail de l'adresse e-mail concernée.
```

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md [hunk 8]
```diff
@@ -89,7 +109,7 @@ Suivez les étapes décrites en cliquant successivement sur chaque onglet ci-des
 > **Etape 2**
 >> Depuis la fenêtre **Options**, dans la colonne de gauche, dirigez-vous dans la catégorie **Traitement automatique** de la section **Courrier** puis cliquez sur `Règles de boîte de réception et de rangement`{.action}. <br><br>
 >> ![emails](images/emails-all-02.png){.thumbnail}<br><br>
->> Cette fenêtre permet de gérer vos redirections mais également d'appliquer des filtres sur l'ensemble des e-mails entrants.<br>
+>> Cette fenêtre permet de gérer vos redirections, mais également d'appliquer des filtres sur l'ensemble des e-mails entrants.<br>
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md]:
```suggestion
>> Cette fenêtre permet de gérer vos redirections mais également d'appliquer des filtres sur l'ensemble des e-mails entrants.<br>
```

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md [hunk 9]
```diff
@@ -188,6 +235,18 @@ La méthode pour créer un alias ou une redirection est strictement la même. Su
 > Lorsque vous choisissez le mode de copie « **Conserver une copie du mail chez OVHcloud** », une redirection de l'adresse e-mail vers elle-même est créée automatiquement dans la liste des redirections, elle matérialise cette copie locale.
 >
 
+### Supprimer une redirection ou un alias sur une offre MX Plan historique ou un nom de domaine sans offre e-mail <a name="del-mxplanlegacy"></a>
+
+Depuis l'onglet `Emails`{.action}, cliquez sur `Gestion des redirections`{.action} à droite de la fenêtre.
+
+Cliquez sur le bouton `...`{.action} à droite de la redirtection concerné, cliquez sur `Supprimer la redirection`{.action}
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md]:
```suggestion
Cliquez sur le bouton `...`{.action} à droite de la redirection concernée puis cliquez sur `Supprimer la redirection`{.action}
```

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md [hunk 10]
```diff
@@ -188,6 +235,18 @@ La méthode pour créer un alias ou une redirection est strictement la même. Su
...
+
+> [!warning]
+>
+> Il n'est pas possible de modifier une redirection ou un alias, il est nécessaire de la supprimer et la recréer.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md]:
```suggestion
> Il n'est pas possible de modifier une redirection ou un alias. Vous devez la supprimer puis la recréer.
```

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md [hunk 11]
```diff
@@ -158,7 +199,13 @@ Cliquez sur le bouton `...`{.action} à droite de l'alias concerné, dans le men
 
 Connectez-vous à votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr) et dirigez-vous dans la section `Web Cloud`. Dirigez-vous dans la sections `Emails`{.action} :
 
-La méthode pour créer un alias ou une redirection est strictement la même. Suivez les étapes décrites en cliquant successivement sur chaque onglet ci-dessous :
+> [!warning]
+>
+> - La méthode pour créer un alias ou une redirection est strictement la même.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md]:
```suggestion
> - La méthode pour créer un alias ou pour créer une redirection est strictement la même.
```

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md [hunk 12]
```diff
@@ -12,15 +12,17 @@ updated: 2024-03-26
 
 ## Objectif
 
-Vous trouverez dans ce guide différentes informations et aides concernant la configuration de vos redirections et alias e-mail, par exemple pour renvoyer des e-mails reçus sur une adresse A vers une adresse B.
+Vous trouverez dans ce guide différentes informations et aides concernant la configuration de vos **redirections** et **alias** e-mail, par exemple pour renvoyer des e-mails reçus sur une adresse A vers une adresse B.
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md]:
```suggestion
Retrouvez dans ce guide différentes informations et aides concernant la configuration de vos **redirections** et **alias** e-mail, par exemple pour renvoyer des e-mails reçus sur une adresse A vers une adresse B.
```

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md [hunk 13]
```diff
@@ -188,6 +235,18 @@ La méthode pour créer un alias ou une redirection est strictement la même. Su
...
+
+> [!warning]
+>
+> Il n'est pas possible de modifier une redirection ou un alias, il est nécessaire de la supprimer et la recréer.
+
+![emails](images/email-del-legacy-redirect01.png){.thumbnail}
+
 ## Aller plus loin
 
 Échangez avec notre communauté d'utilisateurs sur <https://community.ovh.com>.
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.fr-fr.md]:
```suggestion
Pour des prestations spécialisées (référencement, développement, etc), contactez les [partenaires OVHcloud](/links/partner).
 
Si vous souhaitez bénéficier d'une assistance à l'usage et à la configuration de vos solutions OVHcloud, nous vous proposons de consulter nos différentes [offres de support](/links/support).
 
Échangez avec notre [communauté d'utilisateurs](/links/community).
```

**Comment by BrenKt** [comment]
to translate

**Review by bs-ovh — APPROVED** [review]

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.es-es.md [hunk 14]
```diff
@@ -72,7 +92,7 @@ Para estas dos excepciones, solo es posible la redirección. Le invitamos a cons
 
 Las redirecciones no se gestionan desde el área de cliente, sino directamente desde el webmail de la dirección de correo electrónico correspondiente.
 
-Acceda a la dirección <https://www.ovhcloud.com/es-es/mail/>. Introduzca **la dirección de correo** y la **contraseña** para conectarse.
+Acceda a la dirección [webmail](/links/email). Introduzca **la dirección de correo** y la **contraseña** para conectarse.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.es-es.md]:
```suggestion
Acceda a la dirección [webmail](/links/email). Introduzca **la dirección de correo** y la **contraseña** para conectarse.
```
```suggestion
Acceda a la dirección del [webmail](/links/email). Introduzca **la dirección de correo** y la **contraseña** para conectarse.
```

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.es-es.md [hunk 15]
```diff
@@ -100,33 +120,54 @@ Siga los pasos que se describen haciendo clic en cada pestaña:
...
 > ![emails](images/emails-all-07.png){.thumbnail .w-640}
 
+### Eliminar una redirección
+
+Siga los pasos que se indican haciendo clic en cada ficha:
+
+> [!tabs]
+> **Paso 1**
+>>
+>> Una vez que se haya conectado a su dirección de correo electrónico a través del [webmail](https://www.ovhcloud.com/fr/mail/), haga clic en la rueda dentada situada en la esquina superior derecha y seleccione `Options`{.action}.<br><br>
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.es-es.md]: "Options" en Spanish ?

Webmail link to updated using /links

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.es-es.md [hunk 16]
```diff
@@ -186,8 +233,24 @@ El método para crear un alias o una redirección es estrictamente el mismo. Sig
 > [!primary]
 >
 > Al elegir el modo de copia "**Conservar una copia del correo en OVHcloud**", una redirección de la dirección de correo hacia ella se crea automáticamente en la lista de redirecciones, materializando esta copia local.
-> 
+>
+
+### Eliminar una redirección o un alias en un MX Plan histórico o un dominio sin una solución de correo <a name="del-mxplanlegacy"></a>
+
+En la pestaña «Correo electrónico» de {.action}, haga clic en «Gestión de las redirecciones»`{.action} a la derecha de la ventana.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.es-es.md]: formats are wrong

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.it-it.md [hunk 17]
```diff
@@ -76,7 +96,7 @@ Per entrambe le eccezioni, è possibile solo il reindirizzamento. Per maggiori i
 
 La gestione dei reindirizzamenti non avviene tramite lo Spazio Cliente, ma direttamente tramite la webmail dell'indirizzo email in questione.
 
-Accedi all'indirizzo <https://www.ovhcloud.com/it/mail/>. inserendo **indirizzo email** e **password**.
+Accedi alla [webmail](/links/email) . Inserisci **l'indirizzo email** e la **password** per effettuare l'accesso.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.it-it.md]:
```suggestion
Accedi alla [webmail](/links/email). Inserisci **l'indirizzo email** e la **password** per effettuare l'accesso.
```

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.it-it.md [hunk 18]
```diff
@@ -104,26 +124,45 @@ Per seguire gli step descritti, clicca su ciascuna scheda qui sotto:
...
 > Per applicare un **reindirizzamento semplice** (vedi [schema 1](#diagram) all'inizio di questa guida), aggiungi una regola aggiuntiva al tuo **reindirizzamento con copia in locale** da questa finestra. Clicca su `Aggiungi un'azione`{.action} (riquadro 1) e poi su `Sposta, copia o elimina`{.action} e infine clicca su `Elimina il messaggio`{.action}. Questa regola sposta il messaggio direttamente nel cestino, dopo averlo inoltrato all’altro indirizzo.<br><br>
 > ![email](images/emails-all-07.png){.thumbnail .w-640}
 
+### Elimina un reindirizzamento
+
+Segui i passaggi descritti cliccando su ciascuna scheda qui sotto:
+
+> [!tabs]
+> **Step 1**
+>> Una volta connesso al tuo indirizzo email tramite la [webmail](/links/email), clicca sull’icona a forma di ingranaggio in alto a destra e poi su `Options`{.action}.<br><br><br>
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.it-it.md]: Options or Opzioni ?

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.it-it.md [hunk 19]
```diff
@@ -191,6 +236,22 @@ Il metodo per creare un alias o un reindirizzamento è strettamente lo stesso. P
 > Quando scegli la modalità di copia "**Conserva una copia dell'email in OVHcloud**", viene automaticamente creato un reindirizzamento dell'indirizzo email verso se stesso nella lista dei reindirizzamenti, si materializza questa copia locale.
 > 
 
-## Per saperne di più
+### Eliminare un reindirizzamento o un alias su una soluzione MX Plan storica o un dominio senza offerta email <a name="del-mxplanlegacy"></a>
+
+Nella scheda `Email`{.action}, clicca su `Gestione dei reindirizzamenti`{.action} a destra della finestra.
+
+Clicca sul pulsante `...`{.action} a destra del reindirizzamento in questione e poi clicca su `Elimina il reindirizzamento`{.action}
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.it-it.md]:
```suggestion
Clicca sul pulsante `...`{.action} a destra del reindirizzamento in questione e poi clicca su `Elimina il reindirizzamento`{.action}
```
```suggestion
Clicca sul pulsante `...`{.action} a destra del reindirizzamento in questione e poi clicca su `Elimina il reindirizzamento`{.action}.
```

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.pl-pl.md [hunk 20]
```diff
@@ -76,7 +96,7 @@ W przypadku tych dwóch wyjątków możliwe jest tylko przekierowanie. Zapraszam
 
 Zarządzanie przekierowaniami nie odbywa się w Panelu klienta lecz bezpośrednio w interfejsie webmail danego adresu e-mail.
 
-Do zobaczenia na stronie <https://www.ovhcloud.com/pl/mail/>. Podaj **adres e-mail** i **hasło**, aby się zalogować.
+Przejdź do [webmail] (/links/email). Wpisz **adres e-mail** i **hasło**, aby się zalogować.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.pl-pl.md]:
```suggestion
Przejdź do [webmail](/links/email). Wpisz **adres e-mail** i **hasło**, aby się zalogować.
```

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.pl-pl.md [hunk 21]
```diff
@@ -87,7 +107,7 @@ Postępuj zgodnie z instrukcjami podanymi w opisanych krokach, klikając kolejno
 > [!tabs]
 > **Etap 1**
 >>
->> Po zalogowaniu się do Twojego konta e-mail przez [webmail](https://www.ovhcloud.com/pl/mail/) kliknij ikonkę koła zębatego w prawym górnym rogu, a następnie kliknij `Options`{.action}.<br><br>
+>> Po zalogowaniu się do Twojego konta e-mail przez [webmail](/links/email) kliknij ikonkę koła zębatego w prawym górnym rogu, a następnie kliknij `Options`{.action}.<br><br>
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.pl-pl.md]: "Options" probably to translate

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.pl-pl.md [hunk 22]
```diff
@@ -104,26 +124,47 @@ Postępuj zgodnie z instrukcjami podanymi w opisanych krokach, klikając kolejno
...
+Postępuj zgodnie z instrukcjami podanymi w poniższych zakładkach:
+
+> [!tabs]
+> **Etap 1**
+>>
+>> Po zalogowaniu się do swojego konta e-mail przez [webmail](/links/email), w prawym górnym rogu kliknij ikonę koła zębatego, a następnie ikonę `Options`{.action}.<br><br>
+>> ![emails](obrazy/emaile-all-01.png){.thumbnail}<br>
+>>
+> **Etap 2**
+>> W lewej kolumnie okna **Opcje** przejdź do kategorii **Automatyczne przetwarzanie** w sekcji **Poczta** i kliknij `Reguły skrzynki odbiorczej i porządkowania wiadomości {.action}. <br><br>
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.pl-pl.md]: broken formats

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.pl-pl.md [hunk 23]
```diff
@@ -104,26 +124,47 @@ Postępuj zgodnie z instrukcjami podanymi w opisanych krokach, klikając kolejno
...
+>> ![emails](obrazy/owa-redirect-del-01.png){.thumbnail}<br><br>
+>> Znajdziesz okno do zarządzania przekierowaniami i filtrami.<br>
+>>
+> **Etap 3**
+>>
+>> W oknie zarządzania **Reguły skrzynki odbiorczej** kliknij przekierowanie, które chcesz usunąć. Musi ono być podświetlone. Następnie kliknij ikonę kosza<br><br>
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.pl-pl.md]:
```suggestion
>> W oknie zarządzania **Reguły skrzynki odbiorczej** kliknij przekierowanie, które chcesz usunąć. Musi ono być podświetlone. Następnie kliknij ikonę kosza<br><br>
```
```suggestion
>> W oknie zarządzania **Reguły skrzynki odbiorczej** kliknij przekierowanie, które chcesz usunąć. Musi ono być podświetlone. Następnie kliknij ikonę kosza.<br><br>
```

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.pl-pl.md [hunk 24]
```diff
@@ -104,26 +124,47 @@ Postępuj zgodnie z instrukcjami podanymi w opisanych krokach, klikając kolejno
...
+>> ![emails](obrazy/owa-redirect-del-02.png){.thumbnail}<br><br>
+>>
+
 ### Utwórz alias <a name="alias"></a>
 
-Zaloguj się do [Panelu klienta OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.pl/&ovhSubsidiary=pl) i przejdź do sekcji `Web Cloud`. Następnie wybierz menu w zależności od oferty e-mail:
+Zaloguj się do [Panelu klienta OVHcloud](/links/manager) i przejdź do sekcji `Web Cloud`. Następnie wybierz menu w zależności od oferty e-mail:
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.pl-pl.md]:
```suggestion
Zaloguj się do [Panelu klienta OVHcloud](/links/manager) i przejdź do sekcji `Web Cloud`{.action}. Następnie wybierz menu w zależności od oferty e-mail:
```

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.pl-pl.md [hunk 25]
```diff
@@ -192,6 +239,22 @@ Metoda tworzenia aliasu lub przekierowania jest ściśle taka sama. Postępuj zg
 > Jeśli wybierzesz tryb kopii "**Zachowaj kopię e-maila w OVHcloud**", na liście przekierowań automatycznie tworzone jest przekierowanie z adresu e-mail na ten sam adres, tworząc jego kopię lokalną.
 > 
 
-## Sprawdź również
+## Usuń przekierowanie lub alias z oferty MX Plan historyczny lub z domeny bez oferty e-mail <a name="del-maxplanlegacy"></a>
+
+W zakładce `Emaile`{.action} kliknij `Zarządzanie przekierowaniami`{.action} po prawej stronie okna.
+
+Kliknij przycisk`...`{.action} po prawej stronie wybranego przekierowania, a następnie kliknij `Usuń przekierowanie`{.action}
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.pl-pl.md]:
```suggestion
Kliknij przycisk `...`{.action} po prawej stronie wybranego przekierowania, a następnie kliknij `Usuń przekierowanie`{.action}
```

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.pl-pl.md [hunk 26]
```diff
@@ -192,6 +239,22 @@ Metoda tworzenia aliasu lub przekierowania jest ściśle taka sama. Postępuj zg
...
+
+> [!warning]
+>
+> Nie można zmienić przekierowania ani aliasu. Należy ją usunąć, a następnie utworzyć ponownie.
+
+![emaile](obrazy/email-del-legacy-redirect01.png){.thumbnail}
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.pl-pl.md]:
```suggestion
![emaile](images/email-del-legacy-redirect01.png){.thumbnail}
```
Check other images too

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.pt-pt.md [hunk 27]
```diff
@@ -88,7 +108,7 @@ Siga os passos descritos clicando sucessivamente em cada separador abaixo:
 > [!tabs]
 > **Etapa 1**
 >>
->> Uma vez ligado ao seu endereço de e-mail através de [webmail](https://www.ovhcloud.com/pt/mail/), clique na roda dentada no canto superior direito e, a seguir, em `opções`{.action}.<br><br>
+>> Uma vez ligado ao seu endereço de e-mail através de [webmail](/links/email), clique na roda dentada no canto superior direito e, a seguir, em `opções`{.action}.<br><br>
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.pt-pt.md]:
```suggestion
>> Uma vez ligado ao seu endereço de e-mail através de [webmail](/links/email), clique na roda dentada no canto superior direito e, a seguir, em `Opções`{.action}.<br><br>
```

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.pt-pt.md [hunk 28]
```diff
@@ -105,26 +125,47 @@ Siga os passos descritos clicando sucessivamente em cada separador abaixo:
...
 > ![e-mails](images/emails-all-07.png){.thumbnail .w-640}
 
+### Eliminar um reencaminhamento
+
+Siga os passos descritos, clicando sucessivamente em cada separador abaixo:
+
+> [!tabs]
+> **Etapa 1**
+>>
+>> Depois de aceder ao endereço de e-mail através do [webmail](https://www.ovhcloud.com/fr/mail/), clique na roda dentada no canto superior direito e, a seguir, em Opções`{.action}.<br><br>>
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.pt-pt.md:152]: FR link and broken format

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.pt-pt.md [hunk 29]
```diff
@@ -155,19 +196,25 @@ Para adicionar um alias à sua conta de e-mail, siga os passos descritos clicand
...
 
 Clique no botão `...`{.action} à direita do alias em causa, no menu de gestão dos alias. Finalmente, clique em `Eliminar o alias`{.action}
 
 ![emails](images/email-alias04.png){.thumbnail}
 
-#### Criar um reencaminhamento ou um alias numa oferta MX Plan histórica ou num nome de domínio sem oferta de e-mail <a name="mxplanlegacy"></a>
+### Criar um reencaminhamento ou um alias numa oferta MX Plan histórica ou num nome de domínio sem oferta de e-mail <a name="mxplanlegacy"></a>
 
-Aceda à Área de [Cliente OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.pt/&ovhSubsidiary=pt) e dirija-se à secção `Web Cloud`. Dirija-se à secção `Emails`{.action}:
+Aceda à Área de [Cliente OVHcloud](/links/manager) e dirija-se à secção `Web Cloud`. Dirija-se à secção `Emails`{.action}:
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.pt-pt.md:209]:
```suggestion
Aceda à Área de [Cliente OVHcloud](/links/manager) e dirija-se à secção `Web Cloud`{.action}. Dirija-se à secção `Emails`{.action}:
```

File: pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.pt-pt.md [hunk 30]
```diff
@@ -192,6 +239,22 @@ O método para criar um alias ou um reencaminhamento é estritamente o mesmo. Si
 > Quando escolhe o modo de cópia "**Conservar uma cópia do e-mail na OVHcloud**", um reencaminhamento do endereço de e-mail para si próprio é automaticamente criado na lista dos reencaminhamentos, materializa esta cópia local.
 > 
 
-## Quer saber mais?
+### Eliminar um reencaminhamento ou um alias numa oferta MX Plan histórica ou um domínio sem um serviço de e-mail <a name="del-mxplanlegacy"></a>
+
+No separador "E-mails" {.action}, clique em "Gestão dos reencaminhamentos" {.action} à direita da janela.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/common_email_features/feature_redirections/guide.pt-pt.md]: broken formats

**Pull request merged**